demos/dotnet/dotnet/dotnet.cs

```diff
@@ -27,7 +28,10 @@ static void Main(string[] args)
             Log.IsEnabled = true;
         }
 
-        switch (args[0])
+        var commandArgument = args[0];
```

Comment:
At some point we need to rewire this command parsing logic. The current implementation does not really scale past one or two options.